ocs_ci/deployment/deployment.py

```
@@ -221,6 +222,34 @@
+            positive integer (booleans, empty values, non-integers, and
+            non-positive numbers are rejected)
+
+    """
+    if "mon_count" not in config.DEPLOYMENT:
+        return 3
+
+    mon_count = config.DEPLOYMENT["mon_count"]
+    if isinstance(mon_count, bool) or not isinstance(mon_count, int) or mon_count <= 0:
+        raise UnexpectedDeploymentConfiguration(
+            f"DEPLOYMENT.mon_count must be a positive integer, got {mon_count!r}"
+        )
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)